setup.py

```diff
@@ -29,6 +29,11 @@ def get_version() -> str:
     return str(ast.literal_eval(version))
 
 
+DEPENDENCIES = ["jinja2<2.20, >2.0"]
+
+if not sys.platform.startswith("win"):
```

Comment:
It looks like the canonical way to do this is with `platform_system `. https://stackoverflow.com/a/32643122/2893090

Reply:
Just learnt something new! 🙌